examples/asr/notebooks/1_ASR_tutorial_using_NeMo.ipynb

```diff
@@ -405,7 +403,7 @@
     "from ruamel.yaml import YAML\n",
     "\n",
     "# Parse config and pass to model building function\n",
-    "config_path = '../configs/jasper_an4.yaml'\n",
```

Comment:
Should this be changed? The tutorial is located in `examples/asr/notebooks/` but the config is in `examples/asr/configs/`.

Reply:
Oh for some reason while I'm doing wget configs on my machine are stored at examples/asr/notebooks/configs. I will change that back